src/test/java/ArrayMinHeapTest.java

```diff
@@ -33,10 +34,51 @@ void isEmpty_empty_returnsTrue() {
         @Nested
         class WhenSingleton {
 
+            @BeforeEach
+            void addSingleInteger() {
+                classUnderTest.add(10);
+                preState.add(10);
+            }
```

Comment:
You'll still want tests for `add`. When empty/singleton/whatever, adding something returns true. You'd probably also want to check equality, but other tests will be doing that anyways, so you at least want to check the return value.